.ci.yaml

```diff
@@ -1498,6 +1498,28 @@ targets:
       benchmark: "true"
     scheduler: luci
 
+  - name: Linux_android color_filter_cache_perf__e2e_summary
```

Comment:
Per the flaky test policy, you'll need to add bring up: true to not block the tree